src/components/OcSelect.vue

```diff
@@ -97,7 +97,7 @@ For detailed documentation (props, slots, events, etc.), please visit https://vu
 
 ```js
 <template>
-  <div class="uk-width-medium">
+  <div style="width: 300px">
```

Comment:
Not sure what your policies are but some people consider inline style to be the devil's work :trollface:

Reply:
Is this a 1:1 replacement? or a change in behavior?

Reply:
Only happens inside the `<docs>` examples but yeah I've moved it to a dedicated class within the docs' SCSS files now. It's a 1:1 replacement of setting a fixed width for the select box